Description: fix hyphen-used-as-minus-sign
 fix lintian "hyphen-used-as-minus-sign" warnings.

Author: Hideki Yamane <henrich@debian.org>

---
Origin: vendor
Forwarded: no
Last-Update: 2012-07-17

Index: net-snmp-5.7.2~dfsg/man/snmptranslate.1.def
===================================================================
--- net-snmp-5.7.2~dfsg.orig/man/snmptranslate.1.def	2012-10-10 07:28:58.000000000 +0900
+++ net-snmp-5.7.2~dfsg/man/snmptranslate.1.def	2013-01-15 01:19:05.000000000 +0900
@@ -163,22 +163,22 @@
 .nf
 +--system(1)
    |
-   +-- -R-- String    sysDescr(1)
+   +-- \-R-- String    sysDescr(1)
    |        Textual Convention: DisplayString
    |        Size: 0..255
-   +-- -R-- ObjID     sysObjectID(2)
-   +-- -R-- TimeTicks sysUpTime(3)
-   +-- -RW- String    sysContact(4)
+   +-- \-R-- ObjID     sysObjectID(2)
+   +-- \-R-- TimeTicks sysUpTime(3)
+   +-- \-RW- String    sysContact(4)
    |        Textual Convention: DisplayString
    |        Size: 0..255
-   +-- -RW- String    sysName(5)
+   +-- \-RW- String    sysName(5)
    |        Textual Convention: DisplayString
    |        Size: 0..255
-   +-- -RW- String    sysLocation(6)
+   +-- \-RW- String    sysLocation(6)
    |        Textual Convention: DisplayString
    |        Size: 0..255
-   +-- -R-- Integer   sysServices(7)
-   +-- -R-- TimeTicks sysORLastChange(8)
+   +-- \-R-- Integer   sysServices(7)
+   +-- \-R-- TimeTicks sysORLastChange(8)
    |        Textual Convention: TimeStamp
    |
    +--sysORTable(9)
@@ -186,11 +186,11 @@
       +--sysOREntry(1)
          |
          +-- ---- Integer   sysORIndex(1)
-         +-- -R-- ObjID     sysORID(2)
-         +-- -R-- String    sysORDescr(3)
+         +-- \-R-- ObjID     sysORID(2)
+         +-- \-R-- String    sysORDescr(3)
          |        Textual Convention: DisplayString
          |        Size: 0..255
-         +-- -R-- TimeTicks sysORUpTime(4)
+         +-- \-R-- TimeTicks sysORUpTime(4)
                   Textual Convention: TimeStamp
 
 .fi
